ngsi_adapter/src/test/acceptance/features/component/send_data/send_data_api_resource.feature

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+Feature: Sending probe data
+  As a monitoring element (probe) user
+  I want to be able to transform monitoring data from probes to NGSI context attributes
+  so that I can constantly check the status and performance of the cloud infrastructure using NGSI Context Broker.
+
+
+  @happy_path
+  Scenario: Valid probe data is sent to CB using using an existing and well-formed parser
+    Given the probe "qa_probe" and its associated parser "qa_probe_parser"
+    And   the monitored resource with id "qa:192.168.1.2" and type "host"
+    When  I send raw data according to the selected probe
+    Then  the response status code is "200"
+
+
+  Scenario: Valid probe data is sent to CB using a not existing parser
+    Given the probe "qa_probe" and its associated parser "not_existing_parser"
+    And   the monitored resource with id "qa:1234567890" and type "host"
+    When  I send raw data according to the selected probe
+    Then  the response status code is "404"
+
+
+  Scenario Outline: Valid probe data is sent to CB using an existing and well-formed parser, with valid entity ID values.
+    Given the probe "qa_probe" and its associated parser "qa_probe_parser"
+    And   the monitored resource with id "<entity_id>" and type "host"
+    When  I send raw data according to the selected probe
+    Then  the response status code is "200"
+
+    Examples:
+    | entity_id               |
+    | 1                       |
+    | a                       |
+    | A                       |
+    | 1-2                     |
+    | a_b                     |
+    | 1#B                     |
+    | c:4                     |
+    | 12345678                |
+    | 123.4567                |
+    | abc.1234                |
+    | A123B-c1                |
+    | 123_vB12                |
+    | Ab:123.4                |
+    | [STRING_WITH_LENGTH_60] |
+
+
+  Scenario Outline: Valid probe data is sent to CB using an existing and well-formed parser, with valid entity TYPE values.
+    Given the probe "qa_probe" and its associated parser "qa_probe_parser"
+    And   the monitored resource with id "qa:1234567890" and type "<entity_type>"
+    When  I send raw data according to the selected probe
+    Then  the response status code is "200"
+
+    Examples:
+    | entity_type             |
+    | 1                       |
+    | a                       |
+    | A                       |
+    | 1-2                     |
+    | a_b                     |
+    | 1#B                     |
+    | c:4                     |
+    | 12345678                |
+    | 123.4567                |
+    | abc.1234                |
+    | A123B-c1                |
+    | 123_vB12                |
+    | Ab:123.4                |
+    | [STRING_WITH_LENGTH_60] |
+
+
+  Scenario Outline: Valid probe data is sent to CB using an existing and well-formed parser, with valid parser name values.
+    Given the probe "qa_probe" and its associated parser "<parser_name>"
```

Comment:
I don't see the difference between this scenario and the first, happy-path scenario